marketplace-service/src/main/java/com/axonivy/market/entity/productjsonfilecontent/Project.java

```diff
@@ -0,0 +1,13 @@
+package com.axonivy.market.entity.productjsonfilecontent;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Project {
```

Comment:
the same comment for marketplace-service/src/main/java/com/axonivy/market/entity/productjsonfilecontent/Dependency.java